modulefiles/tasks/cheyenne/python_srw.lua

```diff
@@ -2,4 +2,4 @@ unload("python")
 prepend_path("MODULEPATH","/glade/work/epicufsrt/contrib/miniconda3/modulefiles")
 load(pathJoin("miniconda3", os.getenv("miniconda3_ver") or "4.12.0"))
 
-setenv("SRW_ENV", "regional_workflow")
+setenv("SRW_ENV", "workflow_tools")
```

Comment:
Where does this "workflow_tools" module come from? Is it from the unified workflow repository?

Reply:
@mkavulich - `workflow_tools` is a conda environment built under the existing ### miniconda3 installation, and contains conda/python packages as needed. It could be activated in the same way as `regional_workflow`.

Reply:
@mkavulich -
    `workflow_tools `environment that was installed used the following yaml file from the workflow_tools repository:
    https://github.com/ufs-community/workflow-tools/blob/develop/environment.yaml
```
name: workflow_tools
channels:
  - conda-forge
  - defaults
dependencies:
  - python=3.9*
  - boto3=1.22*
  - black
  - f90nml=1.4*
  - jinja2=3.0*
  - jsonschema=4.17*
  - numpy=1.21*
  - pylint
  - pytest
  - pytest-cov=3.*
  - pyyaml=6.0*
  - tox
```
    
   NB: Requirement for python version is set to correspond to the same version in the default miniconda3/4.12.0 installation